src/odemis/driver/test/zeiss_test.py

```diff
@@ -37,15 +37,17 @@
 # arguments used for the creation of basic components
 CONFIG_SCANNER = {"name": "scanner", "role": "ebeam", "hfw_nomag": 1}
 CONFIG_STAGE = {"name": "stage", "role": "stage",
-                "rng": {"x": (5.e-3, 152.e-3), "y": (5.e-3, 152.e-3)},  # skip one axis to see if default works
-                "inverted": ["x"],
-               }
+                "rng": {"x": (5.e-3, 152.e-3),
+                        "y": (5.e-3, 152.e-3),  # skip one axis to see if default works
```

Comment:
The fact is that it is actually present the moment the Stage object is created due to _updatePosition() being fired just before the initialization finished._updatePosition is followed up by _getStagePosition() and in there GetStagePosition6() is used always. So that makes it difficult to test if it is not present in stage.position. I am unsure why "z" is left out here in the testcase in the first place. I moved the comment (which was already there) because I changed the definition of the dict.

Reply:
Yes, indeed. I realised that later when reading further the code. Omitting the axis from "rng" doesn't means it's not present. We need to keep that behaviour (to not break the current systems). So we need a different way to indicate an axis is *not* present.

Reply:
I have changed this back to removing axes in the __init__ if they are not specified through the yaml file. So this way we 'preserve' the behavior the driver currently had in the past.